server/src/test/java/org/opensearch/extensions/ExtensionsManagerTests.java

```diff
@@ -170,7 +171,7 @@ public List<Setting<?>> getExtensionSettings() {
             Collections.emptyList()
         );
         client = new NoOpNodeClient(this.getTestName());
-        identityService = new IdentityService(Settings.EMPTY, threadPool, List.of());
+        identityService = new IdentityService(Settings.EMPTY, mock(Client.class), List.of());
```

Comment:
why not use the `client` we just created?

Reply:
Good catch! I replaced all instances w/ a mock when I went through changing the constructor to IdentityService. I missed that there was a client instantiated directly above this line. Changed this to use the client above.